recipes/recipes/emscripten_emscripten-32/activate.sh

```diff
@@ -1,13 +1,24 @@
 
 if [ -z ${CONDA_FORGE_EMSCRIPTEN_ACTIVATED+x} ]; then
+
     export CONDA_FORGE_EMSCRIPTEN_ACTIVATED=1
 
     export EMSDK_PYTHON=${BUILD_PREFIX}/bin/python3
     export PYTHON=${BUILD_PREFIX}/bin/python3
 
-    emsdk install  3.1.2
-    emsdk activate 3.1.2
-    export CONDA_EMSDK_DIR=$BUILD_PREFIX/lib/python$($PYTHON -c "import sys; print(f'{sys.version_info.major}.{sys.version_info.minor}')")/site-packages/emsdk
+
+
+    if [ -n "$CONDA_EMSDK_DIR" ]; then
```

Comment:
cool! Does this work locally with boa? I am wondering because I thought that most environment variables are removed when boa runs.